java/server/src/org/openqa/selenium/grid/log/LoggingOptions.java

```diff
@@ -102,7 +113,63 @@ private Tracer createTracer() {
     exporters.add(SimpleSpanProcessor.newBuilder(new SpanExporter() {
       @Override
       public ResultCode export(Collection<SpanData> spans) {
-        spans.forEach(span -> LOG.fine(String.valueOf(span)));
+
+        spans.forEach(span -> {
+          LOG.fine(String.valueOf(span));
+
+          String traceId = span.getTraceId().toLowerBase16();
+          String spanId = span.getSpanId().toLowerBase16();
+          Status status = span.getStatus();
+          List<Event> eventList = span.getEvents();
+          eventList.forEach(event -> {
+            Map<String, Object> map = new HashMap<>();
+            map.put("eventTime", event.getEpochNanos());
+            map.put("traceId", traceId);
+            map.put("spanId", spanId);
+            map.put("spanKind", span.getKind().toString());
+            map.put("eventName", event.getName());
+
+            Attributes attributes = event.getAttributes();
+            Map<String, Object> attributeMap = new HashMap<>();
+            attributes.forEach((key, value) -> {
+              Object attributeValue = null;
+              switch (value.getType()) {
```

Comment:
Missed a `default` case

Reply:
This is the exhaustive list of all the datatypes for event attributes, so any value that is passed will match one of the cases, hence no `default` case.